Platform/TigerlakeBoardPkg/CfgData/CfgData_Int_Tglu_Ddr4.dlt

```diff
@@ -51,3 +51,7 @@ PLDSEL_CFG_DATA.PldSelGpio.PinNumber                        | 15
 PLDSEL_CFG_DATA.PldSelGpio.PinPolarity                      | 1
 
 SILICON_CFG_DATA.PchHdaEnable                               | 0x00
+
+# Set 'TxtEnabled' and 'VmxEnable' to '1' TXT Feature enablement
+FEATURES_CFG_DATA.Features.TxtEnabled                       | 0
+MEMORY_CFG_DATA.VmxEnable                                   | 0
```

Comment:
Please remove the "MEMORY_CFG_DATA.VmxEnable | 0" and the Comment from the DLT. Same for the other DLT.

Reason: if VmxEnable is a MUST for TXT, you should directly set  VmxEnable =1 in UpdateFspConfig in Stage1BBoardInitLib.c